rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2018-12-06
+nightly-2019-02-28
```

Comment:
According to https://mexus.github.io/rustup-components-history/ looks like RLS is missing from this version. Maybe we should use 02-27 instead?

Reply:
Yeah, I was trying a later version to see if we still got the compiler errors. Looks like we do :-(